idutils/__init__.py

```diff
@@ -661,6 +669,11 @@ def is_ror(val):
     return ror_regexp.match(val)
 
 
+def is_viaf(val):
```

Comment:
Do you mind adding some tests in https://github.com/inveniosoftware/idutils/blob/master/tests/test_idutils.py#L21

```python
(
        "http://viaf.org/viaf/1234",  # test value
        ["viaf", ], # type identification
        "1234",   # normalised value
        "http://viaf.org/viaf/1234",  # landing page value
    ),
```


There's one thing I'm slightly concerned about is what happens in the ``test_idempotence``:

```
val = "http://viaf.org/viaf/1234"
new_val = normalise_pid(val, "viaf")
assert detect_identifier_schemes(new_val, "viaf") == ["viaf"]
```

Basically if you normalise a viaf, and run a detection on the normalised value, will it still be identified as a viaf. Because it's just an integer, I think pubmed will kick in instead. I think we had a similar issue with GND

Reply:
Ah, that is the reason a gnd is normalized to `gnd:xyz`? I will add some tests and look how to make it safe. Thanks!